etcd/datadog_checks/etcd/data/conf.yaml.example

```diff
@@ -207,6 +207,24 @@ instances:
     #
     # tls_ca_cert: <CA_CERT_PATH>
 
+    ## @param ssl_cert - string - optional
```

Comment:
Can we have a comment about when to use `tls_` keys and when to use `ssl_` keys ?
If these are mutually exclusive group of keys, please add one more comment mentioning the same.

Reply:
`ssl_*` are deprecated and should not be documented. We are keeping them as valid because of retrocompatibility